Automattic/wp-calypso
# WIP Launchpad: Allow user to go back to Add Links

#### Summary
Currently, once the user lands on the Launchpad checklist screen via the LinkInBio flow, they are unable to go back to the **Add Links** task since it's disabled:

https://user-images.githubusercontent.com/20927667/189232830-28d74eee-bda0-4f50-aae1-daa6a21adac8.mov

#### Proposed changes
This PR allows users to revisit the **Add Links** task:

https://user-images.githubusercontent.com/20927667/189232848-836262a5-9dec-44c1-863f-275d2653818c.mov

#### Testing Instructions
1. Go to `http://calypso.localhost:3000/setup/launchpad?flow=link-in-bio&siteSlug=YOUR_SITE_SLUG&notice=purchase-success`
2. You should see that the **Add Links** task is completed but still active
3. Click on the **Add Links** task
4. You should see the Site Editor screen

#### Pre-merge Checklist






Fixes #67430
